bundles/org.openhab.core.transform/src/main/java/org/openhab/core/transform/TransformationService.java

```diff
@@ -32,10 +32,10 @@
 @NonNullByDefault
 public interface TransformationService {
 
-    String SERVICE_PROPERTY_NAME = "openhab.transform";
-    String SERVICE_PROPERTY_LABEL = "openhab.transform.label";
-    String TRANSFORM_FOLDER_NAME = "transform";
-    String TRANSFORM_PROFILE_SCOPE = "transform";
+    public static final String SERVICE_PROPERTY_NAME = "openhab.transform";
+    public static final String SERVICE_PROPERTY_LABEL = "openhab.transform.label";
+    public static final String TRANSFORM_FOLDER_NAME = "transform";
+    public static final String TRANSFORM_PROFILE_SCOPE = "transform";
```

Comment:
I removed these intentionally. Constants in interfaces are always `public static final` (like methods are always `public`).